lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4934,7 +4937,7 @@ namespace {
                                       getOverridableAccessibility(dc),
                                       name, decl->getLoc(), bodyParams,
                                       decl->getLoc(),
-                                      TypeLoc::withoutLoc(elementTy), dc);
+                                      TypeLoc::withoutLoc(elementContextTy),dc);
```

Comment:
Sorry, I didn't mean that this wasn't an improvement. I meant to ask if it's complete and consistent—does anything else need to change? But I'll assume yes.